### PR TITLE
Debian build steps

### PR DESCRIPTION
**Pull request by snadrus**
It depends on a base64-encoded key shared privately to sign the DEB. 
It will use the internal version. 
The output is 2 deb files that are arch-specific to the build environment, have the inbuilt version, are signed, and cover each of opencl and cuda. 

@magik6k I think this is a super-set of the changes you needed to make to get it building. Lets use this instead.

### CODE REVIEW

File: apt/make_debs.go [hunk 1]
```diff
@@ -70,14 +111,20 @@ func part2(base, product, extra string) {
 		OrPanic(os.MkdirAll(base, 0755))
 		OrPanic(copyFile("curio", path.Join(base, "curio")))
 		OrPanic(copyFile("sptool", path.Join(base, "sptool")))
+		base = path.Join(dir, "etc", "systemd", "system")
```
Comment (LexLuthr) [inline apt/make_debs.go:114]: This part is used for user defined custom services. We should use the system default /lib/systemd/system/

File: build/version.go [hunk 2]
```diff
@@ -25,7 +39,12 @@ func BuildTypeString() string {
 	}
 }
 
-const BuildVersion = "1.22.0"
+// Intent: Major.Network.EvenReleases
+var BuildVersionArray = [3]int{1, 22, 0}
```
Comment (LexLuthr) [inline build/version.go:43]: What's the advantage of this array?

**Review by LexLuthr — COMMENTED** [review]
Looks good with a suggestion and a question.

File: build/version.go (hunk 2 above)
Comment (snadrus) [reply build/version.go:43]: elsewhere (make_debs.go) interpret the version. Having a standard of it being numbers makes that interpretation simpler.

**Review by LexLuthr — APPROVED** [review]

**Pull request merged**